wiki/en/en-Walk-Through.md

```diff
@@ -0,0 +1,56 @@
+You can have a look at [Hardware Setup](Hardware-Setup) for more information on how to set up your hardware. If you need help setting up ASIO4All on Windows, have a look at the [ASIO4All Setup guide on the Windows installation page](Installation-for-Windows#setting-up-asio4all).
+
+### Jamulus main window
+If you open Jamulus, you will see a window which looks like this:
+
+![Jamulus main window](https://user-images.githubusercontent.com/20726856/100796017-4cfa4500-3420-11eb-9d35-aa206d392a5a.png)
+
+### 1. Setting up a profile
+To allow others to identify who you are, you should set your profile. To do so, click on "View" (bar on the top)>"My Profile..."
+Now you will see this:
+
+![Musician Profile](https://user-images.githubusercontent.com/20726856/100796411-e590c500-3420-11eb-9b62-e50d02cb009b.png)
+
+Fill at least "Alias/Name" to set your name and close the window.
+
+### 2. Connecting to a server
+Now you can connect to a server. Click on "Connect" in the Jamulus main window, select a genre list (selection box on the top left), select a server, and click on "Connect". Now you should be connected to the server. If you have problems here, they are most likely due to wrongly configured audio gear or network issues. Have a look at the [Troubleshooting page](Client-Troubleshooting).
+### The first Jam
+Congrats! You can now start jamming with Jamulus. The sliders you see on the right work as a mixing desk: everything you change here will change what you hear. If you move a slider down, this user will be quieter, if you move it up, this user will be louder for you. It's that easy! Everything else like "PAN", "SOLO", "GRP", ... are advanced options. You can try them of course!
+
+![Jamulus main mixer connected](https://user-images.githubusercontent.com/20726856/100801241-01e43000-3428-11eb-8d61-e03b5e648971.png)
+
+If you don't want others to hear your audio, click on "Mute Myself" (left to the mixing board) which will stop your audio from being sent to other musicians.
+
+While we're already here: "Settings" opens – of course – the settings window with all the audio settings, and "Chat" the chat window.
+
+More information about the user interface can be found in the [Software Manual](Software-Manual).
+
+## Advanced
+Of course, you can do a lot more with Jamulus, like setting up a private server, doing online concerts, ... Just click through the navigation! <!-- The community publishes guides, tips, tricks on the [community site](#).  -->
+
+## Help and Community
+If you need help or want to contribute to the Jamulus Project, you can have a look at the following links:
+
+### Discussion Forums
+For help, chat, and general discussions about Jamulus: Have a look at the [forums on SourceForge](https://sourceforge.net/p/llcon/discussion/).
+
+### Development and contribution
+If you want to contribute to the Jamulus Software, have a look at the [Jamulus GitHub repository](https://github.com/corrados/jamulus/) and read the contributing file there. 
```

Comment:
The conventions around the use of the definite article in English are weird and strange, and I'm afraid this reads oddly. It should be, "If you want to contribute to the Jamulus project" or "If you want to contribute to Jamulus". "the Jamulus Software" is not a thing :-)

Reply:
The problem is that we have basically two projects:

1. Jamulus (the software)
2. Jamulus Website (this repo)

Reply:
Yes, the "Jamulus project" is taken to include both the software and anything else (they might want to contribute something to the project that isn't software or the repo too - like membership of some group or a certification of some kind)

Reply:
Ok. Just rename it to what you think fits best on the walkthrough branch.